src/components/board/board.ts

```diff
@@ -11,6 +11,7 @@ export interface Tick {
   time: number;
 }
 export type Duration = PBDuration.AsObject;
+export type AnalogValue = pb.ReadAnalogReaderResponse.AsObject;
```

Comment:
I'm assuming since you defined this because users will need to be able to access this. I forgot to mention, please add this type to the `main.ts` file.

Reply:
is ` tick` supposed to be in that file too? I did that change a couple weeks and didnt add it

Reply:
If `tick` and `AnalogValue` are exported types that users have to have access to, then yes.

Reply:
okay will add them both, thanks